### PR TITLE
Ticket 2: Add countdown

### PR DESCRIPTION
**Pull request by yulongkelly**

### CODE REVIEW

File: src/sections/Landing/countdown/index.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    return () => clearInterval(intervalId);
+  }, []);
+
+  const updateRemainingTime = () => {
+    setRemainingTime(getRemainingTimeUntilMsTimestamp(timestampMs));
+  };
+
+  return (
+    <div className="container">
+      <div>T minus</div>
```
Comment (amanda849) [inline src/sections/Landing/countdown/index.jsx]: What's this T minus part here for?

File: src/sections/Landing/Landing.jsx [hunk 2]
```diff
@@ -1,13 +1,15 @@
 import "./Landing.scss";
 import techNovaText from "../../resources/images/techNovaText.svg";
+import CountDown from "./countdown";
 
 const Landing = () => {
   return (
     <div className="banner">
+      <CountDown />
```
Comment (amanda849) [inline src/sections/Landing/Landing.jsx]: Move the countdown between the Technova words + Women+ in Tech Hackathon line and the mailing list line to match the Figma design that Fariha used. Currently, the white words overlap with the white sign.

Comment (amanda849) [reply src/sections/Landing/Landing.jsx]: While you're at it, could you add some space between parts like how it is on Figma?

**Comment by amanda849** [comment]
Can you also add the grey/white border to each box in the countdown and bold the word "amazing"?

File: src/sections/Landing/countdown/index.jsx (hunk 1 above)
Comment (yulongkelly) [reply src/sections/Landing/countdown/index.jsx]: I looked at the wrong design; removed it

**Comment by amanda849** [comment]
LGTM

**Review by amanda849 — APPROVED** [review]

**Pull request merged**